django/chat/static/chat/style.css

```diff
@@ -415,6 +415,11 @@ html[lang="fr"] .chat-sidebar {
   display: none;
 }
 
+.chat-history-section-header {
+  padding: 10px 0 5px;
+  font-weight: 500;
```

Comment:
You could accomplish this with bootstrap classes (specifically, `fw-semibold`) but I actually suggest a thinner and smaller muted style - will push